packages/react-components/react-link/library/src/components/Link/useLinkStyles.styles.ts

```diff
@@ -76,20 +91,21 @@ const useStyles = makeStyles({
   // Overrides when the Link is disabled.
   disabled: {
     textDecorationLine: 'none',
-    color: tokens.colorNeutralForegroundDisabled,
+    color: foregroundCtrlNeutralPrimaryDisabled,
     cursor: 'not-allowed',
 
     ':hover': {
       textDecorationLine: 'none',
-      color: tokens.colorNeutralForegroundDisabled,
+      color: foregroundCtrlNeutralPrimaryDisabled,
     },
 
     ':active': {
       textDecorationLine: 'none',
-      color: tokens.colorNeutralForegroundDisabled,
+      color: foregroundCtrlNeutralPrimaryDisabled,
     },
   },
 
+  // Semantic-tokens does not include inverted tokens, use existing tokens for now.
   inverted: {
     color: tokens.colorBrandForegroundInverted,
```

Comment:
Wondering if we should bring this to the group to figure out what we want our stance on things like this to be. Should it be handled through theme or do we need explicit semantic tokens here as well?

Reply:
Current stance is that inverted tokens should be avoided in semantic to avoid unnecessary complexity, and that a local variant theme or style override (optionally using custom style hooks) should be localized to the intended component (currently inverted tokens are only used in link and toast headers)